release/src/main/scripts/run_rc_validation.sh

```diff
@@ -484,8 +484,240 @@ else
   echo "* Skipping Python Leaderboard & GameStates Validations"
 fi
 
+# Setting up Docker images for multi-language quickstart tests.
+if [[ ("$python_xlang_quickstart" = true) \
```

Comment:
Please call these changes out in https://github.com/apache/beam/blob/master/website/www/site/content/en/contribute/release-guide.md#run-validations-using-run_rc_validationsh -> tasks included